sample/README.md

```diff
@@ -0,0 +1,70 @@
+# Usage - Code sample
+
+This code sample demonstrates the usage of the `go-batcher`.
+
+## Prerequisites
+
+- [Go v1.15+](https://golang.org/)
```

Comment:
@plasne 1.16 requires github.com/mattn/go-ieproxy@v0.0.1. Is it ok to include in go.mod?

Reply:
It is probably better to upgrade the project to 1.16. I will do that tomorrow on this branch.

Reply:
I upgraded to 1.16 and all the dependencies. However, ieproxy is not required nor desired. That is a module for support Internet Explorer proxy settings. Maybe that was required in your particular case, but wouldn't be normal for deploying on Linux - which would be the common platform to use Batcher on.